pkg/operation/botanist/controlplane.go

```diff
@@ -355,21 +377,6 @@ func (b *Botanist) PrepareKubeAPIServerForMigration(ctx context.Context) error {
 	return b.DeleteKubeAPIServer(ctx)
 }
 
-func (b *Botanist) scaleControlPlaneDeploymentsToZero(ctx context.Context, c client.Client) error {
```

Comment:
Why was this function removed? Wasn't it called anywhere?

Reply:
Yes, I didn't see any usages.